tests/documentation/test_files/DocumentationStyleBear/bad_file.py.test.correct

```diff
@@ -1,4 +1,5 @@
 def improper_indents(indents):
+
```

Comment:
Okay so we actually do the violation of PEP-257 but not fully.
For one-liners:
"There's no blank line either before or after the docstring. "
For multi-line docstrings:
"Insert a blank line after all docstrings (one-line or multi-line) that document a class -- generally speaking, the class's methods are separated from each other by a single blank line, and the docstring needs to be offset from the first method by a blank line. "
And Im confused :confused: